### PR TITLE
Added some new interactions for 2020

### PR DESCRIPTION
**Pull request by Doishy**
More to come

### CODE REVIEW

**Comment by Doishy** [comment]
Ready to go pending review.

File: markdown/appendices/resources.md [hunk 1]
```diff
@@ -25,10 +25,25 @@ title: Other Doomsday Resources
 
 - [Legacy Breakfast](https://legacy-breakfast.com/tag/doomsday/)
 - [Eternal Central](https://www.eternalcentral.com/tag/doomsday/)
+- [The Dead Format episode 106 ft. Nevilshute](https://soundcloud.com/the-dead-format/episode-106-doomsday-with-nevilshute-martin-nielsen)
 
 ## Videos
 
+- [DK_TTGL runs a League with Grixis Meandeck (June 2020) - Portugese]
```
Comment (angrybacon) [inline markdown/appendices/resources.md:32]: @Doishy Unsure it's very relevant to add non-english content 🤔

File: markdown/appendices/resources.md [hunk 2]
```diff
@@ -25,10 +25,25 @@ title: Other Doomsday Resources
...
+  (https://www.youtube.com/watch?v=eMgfFlx0_10)
+- [Jarvis Yu ft. Ari Lax do a League with 4C Meandeck (July 2020)]
+  (https://www.youtube.com/watch?v=mn4GYFoVrRA)
 - [Anuraag Das's VOD featuring monkeyscantcry (march 2020)]
   (https://www.twitch.tv/videos/565641017)
 - [d8dk32's top 8 at the Legacy Titanium Series 1K Challenge (august 2017)]
   (https://www.youtube.com/watch?v=VmmR_3pG0Rs&t=38m3s)
+
+## Highlight Reel
```
Comment (angrybacon) [inline markdown/appendices/resources.md:41]: @Doishy I think the Twitch clips are deleted after some time (months). Can we upload these somewhere else ie. authors' YouTube perhaps?

File: markdown/appendices/resources.md [hunk 3]
```diff
@@ -25,10 +25,25 @@ title: Other Doomsday Resources
...
+
+- [Monkeyscantcry shows the best pile to play around Surgical Extraction]
+  (https://www.twitch.tv/videos/564975441)
+- [Romariovidal finishes a league in spectacular fashion]
+  (https://www.twitch.tv/videos/698945172)
+- [Romariovidal builds the "perfect pile" on Esper Meandeck]
```
Comment (angrybacon) [inline markdown/appendices/resources.md:47]: @Doishy This one was fantastic, but the clip is too short :-(

File: markdown/appendices/resources.md (hunk 1 above)
Comment (Doishy) [reply markdown/appendices/resources.md:32]: I would still like it added, it's only a small link but it is a well made video and we are trying to be at least somewhat international :)

File: markdown/appendices/resources.md (hunk 2 above)
Comment (Doishy) [reply markdown/appendices/resources.md:41]: Possibly?
Not sure how we could go about that.

File: markdown/appendices/resources.md (hunk 3 above)
Comment (Doishy) [reply markdown/appendices/resources.md:47]: What do you mean too short?! YOU WANT MOAR?!

File: markdown/appendices/resources.md (hunk 1 above)
Comment (angrybacon) [reply markdown/appendices/resources.md:32]: Works for me 🙌

File: markdown/appendices/resources.md (hunk 3 above)
Comment (angrybacon) [reply markdown/appendices/resources.md:47]: I meant I can't imagine the uninitiated viewer will understand what was going on tbh.

**Pull request merged**